Doc/library/socket.rst

```diff
@@ -450,6 +450,9 @@ Constants
       same way that ``SO_BINDTODEVICE`` is used, but with the index of a
       network interface instead of its name.
 
+   .. versionchanged:: 3.14
+      Added ``IP_RECVERR`` and ``IP_RECVTTL``.
```

Comment:
I think we should document these properly, if only to be able to specify availability.

Reply:
It looks like we don't list any of the specific options, perhaps because there are so many and they are system dependent.